### ISSUE TITLE
Downloading and Installing FTL failed

### ISSUE THREAD

**Issue by Ag3ntSn4k3** [pi-hole/FTL#869]
### Versions

- Pi-hole:  v5.1.2
- AdminLTE:  v5.1.1
- FTL:  v5.1

### Platform

- OS and version:  Ubuntu 18.04.5 LTS

- Platform:  Hyper-V (VM)

### Expected behavior

Update works as design ;-)

### Actual behavior / bug

Download URL not avalible.

### Steps to reproduce

```
  [i] Checking for updates...
  [i] Pi-hole Core:     up to date
  [i] Web Interface:    up to date
  [i] FTL:              update available

  [i] FTL out of date, it will be updated by the installer.

  [✓] Root user check

        .;;,.
        .ccccc:,.
         :cccclll:.      ..,,
          :ccccclll.   ;ooodc
           'ccll:;ll .oooodc
             .;cll.;;looo:.
                 .. ','.
                .',,,,,,'.
              .',,,,,,,,,,.
            .',,,,,,,,,,,,....
          ....''',,,,,,,'.......
        .........  ....  .........
        ..........      ..........
        ..........      ..........
        .........  ....  .........
          ........,,,,,,,'......
            ....',,,,,,,,,,,,.
               .',,,,,,,,,'.
                .',,,,,,'.
                  ..'''.

  [✓] Update local cache of available packages
  [i] Existing PHP installation detected : PHP version 7.2.24-0ubuntu0.18.04.6
  [i] Performing unattended setup, no whiptail dialogs will be displayed
  [✓] Disk space check

  [✓] Checking apt-get for upgraded packages... up to date!

  [i] Installer Dependency checks...
  [✓] Checking for dhcpcd5
  [✓] Checking for git
  [✓] Checking for iproute2
  [✓] Checking for whiptail
  [✓] Checking for dnsutils

  [✓] Supported OS detected
  [i] SELinux not detected
  [i] Performing reconfiguration, skipping download of local repos
  [✓] Resetting repository within /etc/.pihole...
  [✓] Resetting repository within /var/www/html/admin...
  [i] Main Dependency checks...
  [✓] Checking for cron
  [✓] Checking for curl
  [✓] Checking for iputils-ping
  [✓] Checking for lsof
  [✓] Checking for netcat
  [✓] Checking for psmisc
  [✓] Checking for sudo
  [✓] Checking for unzip
  [✓] Checking for wget
  [✓] Checking for idn2
  [✓] Checking for sqlite3
  [✓] Checking for libcap2-bin
  [✓] Checking for dns-root-data
  [✓] Checking for libcap2
  [✓] Checking for lighttpd
  [✓] Checking for php7.2-common
  [✓] Checking for php7.2-cgi
  [✓] Checking for php7.2-sqlite3
  [✓] Checking for php7.2-xml
  [✓] Checking for php7.2-intl

  [✓] Enabling lighttpd service to start on reboot...
  [✓] Checking for user 'pihole'

  [i] FTL Checks...

  [✓] Detected x86_64 architecture
  [i] Checking for existing FTL binary...
  [i] Downloading and Installing FTL...curl: (6) Could not resolve host: github-production-release-asset-2e65be.s3.amazonaws.com
  [✗] Downloading and Installing FTL
  Error: URL https://github.com/pi-hole/ftl/releases/latest/download/pihole-FTL-linux-x86_64 not found
  [✗] FTL Engine not installed

  Unable to complete update, please contact Pi-hole Support
```

**Comment by ibigbug** [comment]
sorry but why is this closed? got same thing here

**Comment by yubiuser** [comment]
@ibigbug 

Ag3ntSn4k3 closed it - maybe the issue doesn't exist for the user anymore. If you have the same/similar issue, please open a new issue.

**Comment by DL6ER** [comment]
> Could not resolve host: github-production-release-asset-2e65be.s3.amazonaws.com

Is likely an issue outside of Pi-hole. We don't force the DNS server to be the Pi-hole itself **on** the Pi-hole to avoid loosing DNS abilities when an update fails.

**Comment by Ag3ntSn4k3** [comment]
Hello everyone,
i had the problem that i could not reach the page "github-production-release-asset-2e65be.s3.amazonaws.com".
switching off the pihole did not help. Only after a reboot and deactivating the pihole it worked.

If it was a timing problem that the URL was not reachable, I can't say.

After that I closed the issue.
